proto/ibc/applications/middleware/fee/v1/tx.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package ibc.applications.middleware.fee.v1;
+
+import "gogoproto/gogo.proto";
+import "ibc/applications/middleware/fee/v1/fee.proto";
+import "ibc/core/channel/v1/channel.proto";
+option go_package = "github.com/cosmos/ibc-go/modules/apps/29-fee/types";
+
+// Msg defines the ibc/fee Msg service.
+service Msg {
+  // RegisterCounterPartyAddress defines a rpc handler method for MsgRegisterCounterPartyAddress
```

Comment:
```suggestion
  // RegisterCounterpartyAddress defines a rpc handler method for MsgRegisterCounterpartyAddress
```

Reply:
Maybe I'm missing something obvious but what is the change here? 

``` 
// RegisterCounterPartyAddress defines a rpc handler method for MsgRegisterCounterPartyAddress 
```

This is already the comment

Reply:
Ahh its the capital P